### ISSUE TITLE
tb from browser example

### ISSUE THREAD

**Issue by pjz** [jstasiak/python-zeroconf#7]
I sucked down a copy of examples/browser.py and did a pi
```
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/zeroconf.py", line 833, in run
    self.readers[socket_].handle_read()
  File "/usr/local/lib/python2.7/dist-packages/zeroconf.py", line 889, in handle_read
    msg = DNSIncoming(data)
  File "/usr/local/lib/python2.7/dist-packages/zeroconf.py", line 458, in __init__
    self.readOthers()
  File "/usr/local/lib/python2.7/dist-packages/zeroconf.py", line 505, in readOthers
    domain = self.readName()
  File "/usr/local/lib/python2.7/dist-packages/zeroconf.py", line 554, in readName
    length = byte_ord(self.data[off])
IndexError: string index out of range
```

I can't figure out how to get pdb to attach to it since it happens in an Engine thread, so I'll leave it to you to suggest a fix.  Or more debugging steps - I'm good with whichever.

**Comment by jstasiak** [comment]
Related: #17

**Comment by stephenrauch** [comment]
Closed via #68
